homeassistant/components/rituals_perfume_genie/entity.py

```diff
@@ -22,22 +34,27 @@ def __init__(self, diffuser, coordinator, entity_suffix):
         self._hubname = self.coordinator.data[HUB][ATTRIBUTES][ROOMNAME]
 
     @property
-    def unique_id(self):
+    def unique_id(self) -> str:
         """Return the unique ID of the entity."""
         return f"{self._hublot}{self._entity_suffix}"
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the entity."""
         return f"{self._hubname}{self._entity_suffix}"
 
     @property
-    def device_info(self):
+    def available(self) -> bool:
+        """Return if the entity is available."""
+        return self.coordinator.data[HUB][STATUS] == AVAILABLE_STATE
```

Comment:
We probably want to check `super().available` too.

https://github.com/home-assistant/core/blob/6604614c399610cc40341e99da278627499b35a1/homeassistant/helpers/update_coordinator.py#L303-L306